src/tasks/tasks.py

```diff
@@ -311,6 +320,7 @@ class RankingTask(Task):
 
 
 @register_task('sst', rel_path='SST-2/')
+@register_task('sst-alt', rel_path='SST-2/')
```

Comment:
It's to merge all the BERT-related changes together, but on second thought I'll extract this out since it's not essential for BERT integration and this is already a monster PR